go.mod

```diff
@@ -1,13 +1,15 @@
 module github.com/libp2p/go-libp2p-pubsub
 
-go 1.22
+go 1.22.0
+
+toolchain go1.23.3
```

Comment:
Not necessary.

Reply:
It was automatically changed because of my local versions used. Removed now.

Reply:
https://github.com/golang/go/issues/65847